src/indexer/broker/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use tokio::sync::broadcast;
+
+mod processor_connection;
+mod server;
+mod types;
+mod util;
+
+#[tokio::main]
+async fn main() -> Result<(), ()> {
+    env_logger::init();
+
+    let processor_url = std::env::var("PROCESSOR_WS_URL").unwrap();
+
+    let (tx, _) = broadcast::channel(2048);
```

Comment:
How about making this an env var?

Just in case AWS wants something special

Reply:
This is just the size of the buffer of the broadcast channel (i.e. it will retain up to 2048 unread messages). It has little to do with AWS. I can put it as a `const` so that we don't have magic numbers hanging around.

Reply:
Addressing in ECO-2115